packages/obonode/obojobo-chunks-figure/image-properties-modal.js

```diff
@@ -59,15 +81,46 @@ class ImageProperties extends React.Component {
 			>
 				<div className="image-properties">
 					<label htmlFor="obojobo-draft--chunks--figure--url">URL:</label>
-					<input
-						type="text"
-						id="obojobo-draft--chunks--figure--url"
-						value={this.state.url || ''}
-						onChange={this.handleURLTextChange.bind(this)}
-						ref={this.inputRef}
-						size="50"
-						placeholder="Web Address of the Image"
-					/>
+					<div className="flex-container">
+						<div>
+							<input
+								type="text"
+								id="obojobo-draft--chunks--figure--url"
+								value={this.state.urlInputText || ''}
+								onChange={this.handleURLTextChange.bind(this)}
+								ref={this.inputRef}
+								size="50"
+								placeholder="Web Address of the Image"
+							/>
```

Comment:
I was able to paste in a data URL which showed up in the image preview but fails to upload.

An example URL: `data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxMTEhUTExMWFhUXFxgaGBgYGR0fHxofGh0aGRsaGhodICggGB0lHRgdITEhJSkrLi4uHR8zODMtNygtLisBCgoKDg0OGhAQGy8lICUtLS0tLS0tLS0tLS0tLS0tLS0tLS8tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLf/AABEIALcBEwMBIgACEQEDEQH/xAAbAAABBQEBAAAAAAAAAAAAAAAFAAIDBAYBB//EAEQQAAEDAgQCBwUGAwcDBQEAAAECAxEAIQQSMUEFURMiYXGBkaEGMrHB0SNCUnLh8BRiggcVM0OSovEWY7JTtMLS8jT/xAAaAQADAQEBAQAAAAAAAAAAAAAAAQIDBAUG/8QAMBEAAgEDAwIEAwgDAAAAAAAAAAECAxEhBBIxQVETImGRBYHRFEJScaGx4fAyQ8H/2gAMAwEAAhEDEQA/ANJ7ftZX0W/yxBvso/C1edLdOdRgXOaw0N/IX07K2/8AaDiluPNOCQksIUn+qSZ7ZOlYz78bkp8hXPiODfpkI8DxuXFIWbJQkAm/4TPxIrQLx6en6ipStKhuIMEmLaGJgbk2rMYFQDiiQOtNvKrwxGUGMpIkgjbkbmARPoJrOdOMuQRziOIT0GRIEpzZoB1UuD/tjS2tU+GKRKi6ohOUpSE7mIhRmyZINwZttNRY18fajmbf60mR4CncExiQpwKAJiJV93fMPL0ot3CUbqyBDa3EtkBSvfbCRPVJBKpvygeYvVnDpcfX0iS2txZyhBylRTBGYok5DoLaWvGtriPRpCsgzFaYSM1myVG+Ub3O+6RcCrnAm1vPMlspZXKUFSVRAuLAWiAYGutUnlM59jV0wVjPZ5z39CEIKkr6pCrHKAq6pCgqe/kaNPYFlK0ENELShhRylZK8ySMoCiQBmAmAeQ7PTONcHZcSQktlxSS2VrV1lApgAqmVb86rYvg6WkFS3ktISlCR0ZiICUmCokiR1Zmw7a1cSUjLcSwmC+y6INpcXAVKiCiyDJlQH3hY6wa1vCFYPoQChMQoKEyNCpedRUdRqTbasbxfE53yqLmBmIJ0AA6xmVdQX8a03ssw0s9ItKQEkmFalRElQB7z3yRWHjefbbBWb2Rc4jwnDZg4UJcaUBIDcpTKpzhWYQSopkXskGBc1Bj/AGEYfQFNjo1lIhZKjl1jq5hmsYuYsDerftBiGsQyprpMiIBgGDAvpEADWLzA7jYw3EkoZSDiGlHIAAFJSSqwzTmsZkx61tFxcsFtO2TzX+6oZx7yjnTh0KSlRHvKKiArUD3QPMWOlYLho6TM50gGUg5VA9Ym2UbSbG8WnkY0fF8Ys4ToM8fa51p/EUoUk5iDcDLMGRMHlWfwDiWkkLSACSqL7DSdbxfvqcWwJ8jsMoEZQCAJm1p211P73o3hGD0RScuVBGYyYTEm282JAvVThgStlCjm6RSlZgi+ROxUDuZsOQPKj3CsXlUEEhaTEiw1zBMCRlMzuLzuahvsUuLE7TTyh0YZKYVC1FvrXgAKMBUBUzm+UVpeFYVuHGyEKIa+9BEgSFJ8VSCLgZTuRWX4VjnM68rhTnBzKMEkEzckTNhejWGxikKJ6W6UiNBA90C14HrUuUU72IvYWNwJw6860lbR6QIKjmgkqAFlXyzMmI5Tcz8PR9kF9IEoC1BCVmxmOsrtCconWZ5VzjPHVuslrqySCDckHzvoaF8GKMgDnWAHVSo3B2gi8j13qJpN3iaJmi4M100tNIMBRzrJJIVEEkqETYAA6C+um1wnCG0pIKEknXeOwE37Z3Mms5wT2hQ03lyzJUom+6iTtR/BcbYUkfaAGLzNvOuii47c8hNu+DN8VZU7jGsOpIyIABg2UDBJjtSPjTcdwvpGkQSvMhByrUpQnmCTqKstYps41x8qAQJAM2JCMvxmuktJSMr6CQkJAmAIm8xvNRO7yi0zxZxMPqBH31AiNwTa2lHsMRkWOeYW15aVmuK4hQecIicyjIJvKv1sKMcKxqlCCORNo2gmPnUTXUpPoDmjh8rKVFa3swsBZKVGFIGhKpKjftrWcP4WwhKswKnViVFSiSCNRMx/xBrHjBlCicoVBUFKI1znMMhjSxvqDOxFFcOklWYqPWSjIQDYaZjvJiYrabbWGckk1g1PEMI0hKVMwFApBQr7wHd+/GiHDsS2tJCkpyhMGBaeUkmZ7ht30E6Q5QIUoqChBVsfeIOoBkaGPSh+EeUnqJlCbyADbQpjb9J5VxR3VE4s6IqzTRtcXkyZQM0jwPlWXdxrRBWW0+8VGR5zPZFWcPishhS9BANtT2anVJ8aF4lKujyzJKrg8iQmwP8AMdP1qKFJ3dyq2bFhTbbnXyt9YDUnlvSqsweqIWAOVvpSro2nNaXYkx/EFOobB/y2wjaerpoL+vrQZ5Qm8n9z8agbKygaXnUgWBPM69nlTVpUdLEQPWK3Olq5KjFggk3vy0Og/wCaixOIIITME78+21qTGDUQdJF4J1i+k9aw9KjGFm/Z5DlQmRZj8TiUqBjmI9I9LVWafIJ7RTVNEwE6iI+c761YwvC3FEwhRAF4SToLCRVFLIx7FSImlg8SpCgUEhU2i/d2VfPs+5N0mNLwCTHI32qYcDcBkhKDP4ufZcxQ7FqjN8JhrhaIlQWApKJUrPe+103JJ03nURVxzHtuAgqKyfd2iOrAAFpKZttQ9fs+8sdSDYZiAonQaQkbz51JhPZV4IGac38zaxHfmtvPhXP4MnkydGa5RdQ2F+6sydBJIEEi+0RfU37qZhsepJuVagTPdECdRHxqwjhJTErbEGQM8DlBB7PnSVwqWykYjDpJMyV3BgDWKnwat+BbGuSHF8VKUqKZWCJi5gSJ8NJn1piW31t3IFwbn3Yvrp2R3iqo9nV5pS/hyM18roBIOxteraeB4gBICkqAP3FJJiQd4nSfOrVCaKjbqV+M8MQhKT1SpXVWlPIWStU7zlTt729eeYpBSFDUlUeEm/j9a9D4lwrF5QehXKU2hMi5BmByImNqzeIwZ6UBaTMAgEEXtbn/AM1dOMo8iccEnBsO6nKcsEhMWMG2ZMjQhSVGN9e+iiMVnWSUAGSCLEzJJmZkSsjyqLibHXzA6AaRaLiI20Gg3qRhCilIESMwkReQSBIuZHzo25uKPce03fnHdOp3PdVgtEkA2mRpy63ypmIbhAVmAKzEX2sO7Se80wvmBOs69sGRfvjxppY5Hdhd1lpKRcm47I0GnP8ASqKmBKgk2AOvZ9J9KruOk3uACo22jt11j9mmuOq60axFtYsSJ5Tas0pJ5YJhvCvAZUnrkmJ0uZg7/sU5x49CMp/mMzCvkNDQHAuHNvlnXsg1fGNBTG0Jt2akfvtqNjUrhYvF37JMdWYMkQL/ABmNe2hjrym0OZlkkJBATy95cdoFTOuhaVBMCYPdNkgec8791Bg6vPJVNiFHdQJGYWvv6Vai+RtMDMYVIcKUZlHMMupPWkpkbdUCeR5RFG8LCU5rA2MHYWJn1v2GqWLcCEXUnMLJiQogxJI2UCbd6ZsLxvY0qgDWAFWsbSpWljJOYHtjeL/yyNNdAxhghQczJlIF4MEGTPkCDGm1NwbKm0qAAIzJkdg0T+UwT5dtCcJiAFEXuQTA7QdOYA/e5hniaUhQXBlAgc+U8j1iI5EUSi2iXyMU0G8hFkiwAIkBUGwJnYSRrQnArylWWYzRzgTEkxcx2b0WOIbdSQmxVGv9dgeUG1B8I5LqoskSqBO1hPO3wqYYbuUHmnRkkj3iD5GY8vjVZpWZCVJtBAv2mQDNiL+FD0uFEhJJSYUUiNriJ8BarmExYsnLYiZjYxPqR61UYpO6HJ3JCwDe3nFKuHDqOkxtf60qoiyALruVMAmJzachGvjEVxnECBmuASd7+P6VZe4OCB9qAYuYNzczE228u2mL4IIy9OO3q/rWmxj3pDEuJJJGgNhO2sT46/CrIdbkkqGXLmWeQGoHoBfVVPZ4IkJKelSZ3y6RPb+4odxLAJRlaCswWOkcMRDaJEb+8qfSkoO+SXINcDw7rsrw6GkJTA6RYzqBgGcyoy2I0RvrRF9/GoWGzilkwTbJAgAx/h9vrV32bYQnDovCoKlA2uesYUOWg7qDPvrU6ViSLg9xvr3RVKayyG28XNJwbHuOdG26rOFvBtRgGxBvykEa99E+G4dTgCulU0FXDbQQgJB0GbLnJjfNQbgDhK2SYnpgfJO/KivDFwhB/lT8BXVRSlG7CvVkoxinjIV/6faV7ynV/ndcV6FUVTxHs7hkXLLfihJ9SJosziRBPZNU8diwoWrWKdzkbwUEYDC/+kgHsQn6Vx/BNpiEpj8o+lVEunpDexB9K7icSCIBkCL1tFO5k5F5rCtgTlb8UD6VdbwbZHWZZVbdpP0oGxiSDBNqO4ddgZonGwKRXb4bhibYdtJnVCQk+aYogng7ahCVvJkW+1Wof6FlSfSo3EXkVZaUdQazlFFRm0YP2m4d0LgQoTmHUVlSkKULlCsoCQdCCAJnxAs/Z+6b7i5va08hmmvTPajhIxmFUkRmiUnkoe6fkewmvK0LWuW1pAebJC0q1JFpnnYA87GuKtSvlHVSqO9mTvYkpE8lC3MzPK03oa64VHNOp8IBmQOfbVh3BOEC0lJuZTeBHmKh/u1xIjLbvHYJ18a5bNGz5LDL5CjMwBPZaD9KmadzKmd+yNzr6wfWq4wygSmCNYMW0iPh604yCkKSYzahJ31112vt4UOI2rllpJ6SRBBI00uFm3MC1ceMZyICwLjYxJ23t+5qsl0gxAASgAT8p2pYjF9eAbQuZgyMwknyjwpbXcqw/A4o6qi8EgqERYC496BeB8qdiSohbirQBBMwZzyLaz8tqE4JcZjPMx8Rfvrr6yEEKJibX0BBkx2zfupyV+AaIXFJdKSq0KjeEyEqG8CCufKTUGJc91CVdQx0atwoa8gRIHr4wOiRBj7oA8BfyjntSOIIASpZjqwDFjIvNoPjtVW7AlZl7DLzLJIAVE7QFETbaLxXHDKZm0RvcnKBpzuaG4YkKSCDfcTbSSe+CfCiK0pumRc2B319I5cqawBdwTsAk+990W7NDHfH61GxiMt02TEFUQfvbXixme6q7TxEiIKbjyBItvNvWml/ICTcnW+k9v718paGXi+OqBrEK5jY/wDP60wJhQUmJumJPn6fCqqFBRiBeBm8/mKnxAzAn3SIjWJmfK8UJWJCKcakC6oO+lKqiHQQJQgncnX1pU8FBZOBkg51GbCQnbX7t6crhShm+0VOv3e8xbt0oih5ZR7wmdim9qSMUoKhQvlmerzEyN9KjeJFdWBbTlKlKgi5hHn7tgO2spiklSXHlJILrgbQDs2gDbukeFavCuZ1S6okaXunlBTy2miL/BW1QUpCuSReeeVBN7boUaqE1az59Tarppq23JnMMt0SFSm33rehNScJcQF/aSU5pgWNjsruAE7Vi8ct1paw50iIUZzBSd7a1xniCrdckd810qjDba5xvcmeqcPQlL6AiMnSOFMcsoi5uY0k8qJ8P/w0A/gT8BWO9h3pW2M0guLCQdswQI8VK9a2uFUno2rdYJSCSRsLgjvrooramiNQvLB+j/dk2GSSFJGu3ZVDGJKDOYdtHcGkZp27L0F9ouI4VskqeSNjfcbd9bRaucrTfAPy5juNdKjLKxcDeuHHiApCEmYgrdbRM7wTPpVR999y38Qw2n/tqSo+fWJ8IqnVigVGb6BFTZEGrf8AGwANDGt6z6OGp+/jVK8XfkI9Knb4PhNVHOe3OfikUvGiy/stXpF+xrcDiZAkSCNUmautLynmKzeFW2iA2lwACISm3gCb99TY/jnRoKuheOl4SBJMCTmnflUucO5S01b8LNZhHMpjY1nfapTLTiemaSGnRlGICRmaWIyFSvwkWuYte00FXx3GLT1klr8JQkKJ7SSYHcJobimHHv8AFexaxukwEnvSDGtYynHobR0Vf8LB3HynChWfPm91OUgpv1hB1gi4Mm1tRVEY0mCVpuQdefdy5VZxXCoSUQVoMdRWVOmgtcXiranARGVANoBtEEWGvKsKk49DdUKsFeaaKQeeT1imfEc9TXf4l2wAVy5W5zpU76EGR0SJHLXzi/d204ts5YyKSZ2URp3GNKzVQTiVVcSckdVwCdgbAdop/wDfO2QDckp2+VNwuEQP8xYHICd5Fyk1VbBEqDyc3JevlYiq3IVrF1OKQdkzYaAxNz9aS3Wb5gDaTNu6k3gnlgLCmlDSCkiZ7b9u9VcZglSQppHug9VRsDO0C8jelujwG1kraGCqejE88ygLCNARFqjxGBwyzJzSO2f/ACmfGqaUJEgtuA6QlQiZHbUXRpz3LqYG4PrAI3oSXIZQTwnCmpAStczOo5/ln4V1/giJzB0jW0cye3tqk0pEyHkk21IGnYRUasykyHAdI28LE/uae1NhctHhWa/SCbycuxCZm8fdFu+oneBuKEAoMRHhrtznemsFY1IuOZBHfVhh9wEgJt+YHlreaNsUGRiOFuoF4m3I8z4XNcRg3PvCyu4kx2b0lYl2VDKoqnUXGtyCNR21xPEFA3jlJHneKShHuF2uhVVhXJshfl+lKro4kN1UqW0eRyeJKjv2t9LVIOJKBOhkbgfS1CjguRNT4ZrKQVSUjasFBMakkHGcYgpyOiU/jT7yf/sPWrhStKJbVnRzTcf1JOh7wKAYXHtrJbywpScyT3KIUnyvVplSkHMlRSeY+Y37jS2u9nwevDjdEO4T2qUkDOkzEQu8cwFe8BPIkU1vB4XEhTnRtEnUKQhMf1oAUnvIA7aoOYptwFLgyH8aQcvepGqe9J8KDv8ACH2SHBJSfdcbPLtFxaq3yg8cG1OhSqrbLEvXr/fQtcESG1QUnKHHJSYslYACTPKLztWideUb5sVfT7dzL2QAqKzDfF83+MkLH409VY/qFljsUJ7RWx4JxNpwDIYIABgf+aN+9JNVTq7nZs6qmnhSgkocd8+oOfwDqgrqrmDBK5IOxGY6+FC2+FEqGbDrmRmWRJ7byd+VbPFgoTmHu/y3SPmnutQx3GzoddIrZpcXJpKLe5QV+CDD8OSRAGVX86T/APE/SrH9yk/5iQfyE/FdR4biaE++rTerg40ypQQAVX1JA+NK9O3JclWTwsDcL7OLJlTuXuCQe8amjOF4ElNxiXe0SkR3gpkeNZnH8WaPVQFeCj6wYme2L0PUsrNlmeSpv/UDf0FQ6kY8IPs9aorylb5I9KS422AC4m1pJE+JqLEcVw8ELWhQ3EhXoJrzpsr92OqdDa/9VxHjXFYaNYJO0yfIXqJatrhGS+Fxv5pmix3GcClMjCoJ5dGj4waCLx6F3Q00mT91tMjvJ0qHoScsjvm3kae5gVpIKYHaNf8ATFYyrVJHZDTUYYvn1Z1KQoCUpteQB6RqfpS6LQXv4z21eYYChIjTWDa2hyyRUycCqBCQZuCCfMHUXrPzkycOGB14QHnr2j4VzEYEEe9lO+sDTvPbWhThiFSpKhaCIknaJg1WOGTMKISZvIG+sGbQdq0ju5OOdCjPlewMe4ItLZdBJEXylKgLb5SY8apjCwCSU2Fr+Oka350WxZ6M3RqSLwJHZ4xVFtaQFJKJUTIM39LEVbnbBzv4W5K8WCkoTJBiZj9kUxKSAqCZ/MTNu+p3wCSYg8gfkd/Klh1ZZg3MSCAPgaXiX4OWr8O1FPO269DuFayt5tCTmIidL3k3mp8LjnFk2QTESoEd03+VoqNalWBCbGbEeWnbXGXxqUKPLW/yq7nC1Z2ZD/DIAJKZOa+kT2RtApJaZzEZAoRsmY8wJoil/DlvKpRsNDbstpY1VDLarJVAm0Ek951iquxYKjGBZJJJyxJSBI7Ysac3hGzbOtOgMm+0xmEetPew2WVZwY1SRGhG3rUrmKSqCUAi4EW8YvTcmuoKNyPHcD6MjI+pQUZghJ5XsKpBL6TlCknlqO3nFXELTmylNtcvKeVRFpBFydBP6yKFJ2yFuwIXi35ukE88x+lKi67k9ae2P1FKn4gbH3LamKarD1bKajWBW7ijEzmPw/RPYbLrmUJ7FbHuzG/LuoyFTUOMal7Cqj/PQkz/ADqSKJ8UwJacU396TbneDrvINu6NRUThi6O7Tanb5JfIo5qfh8Ytoy2qNyk3SqPxJ0PxqEmmE1zu6eD2I7ZqzCuNZw7shcYZ4jnLap3zfdmdFeKqC43g2IYM5TpIUg7fiEXjtFu2plLJBBMg3vflpOlhFXOFYp9sQ0cyJu2uCjvgkZD/ADAg1DUXzhm1GrWpceZdn9fqRcN45iGoUhxRkSQq49aMpxGHxPvDoXD94e4T2j7v7vQ9xLLxyqH8O9pCj1FTyVsTO+us1Wxza2RBSRA8OUg7jtFqxm6kH3X99jpp1KdV+TEvZ/yi7jsG4yrriAdFA2Pcd/GqhaJgyI7ufdXeGe1S0jI6AtB1SRRL+DbdGfDK1/yyb/0nfuMGr2p5R1qcoYmrevT+Aew2SDrM2GwnfWuZVqN5MbkyBeNdhXXUwcqgUnkfmKeywZEJJVNv0HOoNr9QrgeHuIAKUKUFfeg5eeo94d1F8HhTkUJSCDBKRG5FvK9QYdHVCgQEpurOlszG2W8XtRjAoXIUlIAI2MDxETpaRXVCkeNXqyfNiVtCVEB0BMAWIN+2frUzXAkZ8+axMwJHjrrFqtF+B1sp7/XWqqnED3FhP8uaR5AyK6fCj1R5+6b4dhuL4GlJLjSl5pBKMwAUBsDFjyJ8daM4RtCBblFwJ2mYFzbWgx4inMEqJSqCYg3AsSLTFJXE2+Z8j86tRguBTpVJpJ3YcWhG0DuFCcYTBABVr2xVVfGEjZXp9aGYv2oaR7ykp/M4BQ9rHT081yc4hhcrWYoU4sSAkX10tokW1T5Gs09w95QzFGRINoGnK8Se+jKfalK/8IdIf5EOO/8AgKixnF8QblhaRzcbQ1/7gprnnRUup3UtSqPLj83/AMMw60ArrKk8+fnUjbaj7qVRoCBM1YxPFFpkFeHbPJTwn/Th0L+NE/ZjAtY0rLmJWpTcSG0qCYNgCXE51GZuMtY/ZGa1PjVCK5v+WfogP/CkDrQB/MR8JtUS3GkCA8kRtf0JF69Nw/sDhYktKVN+sc0+Di1AeAFdRw9prMllvo8oGmVII3sgAVrHTd2eTqfilGt/rv68fU82axB16qha08u2oC8m8ovm1B0AiU7TWl9qODNoxBQQhCihKpCgJJmbb3BM1lOigkJXcTIO0TMeVYvyto81xbSZG+oHQ27SdNvSq7QMmD4GpsQtRAJAPKKrKAJuDpeaA4JEBaVTPLznUmrDbyhIUkmQdDvr5VRzDkR56bVKh1QEJPO0fuapsEjgK+Y8QflSqcLXypU9xNg5UblPzVE8Qa7GYlXEOnbUXHfsfOl/aHxYjEtOogocbDgE6hca7pUCFDvm0GK44isxxzDOZpAlMWj50r2FbJocFi0PpzJPW7bSeSuSvQ+tcVbWsVhHXWlZkjvB0PYfrtWzwONQ+m5g6SdQdgrmOR//ADWU4KWUehptVsdpcD6Hs+2bmHdPRJBSOqqYOYb7aW5irr4KQQdRP1rGNsyJ5mohBN5OvV6mUIrb1PXX+gx+FDzKAFD3kjW2308OVZ7hvHXGepAda3bc2/Ir3kH07KG+x+LcwrwzWbcgK5A7E91EPaTCBDyin3VSoRsfvDz+NTVvB3R3aPw9TT83T9PmXXeFYbE9bCL6Nw6sOQD/AEnRY7vKgCg6yspIKFCxB+YpoPOi+BdcxDrLTis4C0AFV1BIMlM8orJqMuMM9CE61HE/NHu+V9Qhw3GPryh7CvuCUwoMOKVlOsEC+0TWmXhXUZFs4HEJTMKLpbRIIISOsuQc2XbSb1r+H8NGZKokGPjRTF8OSQoZRBSREd/1rrVNI+Zq/FK1R8JLsrmCbwuLPuMMJPI4kE+TaFGrCeA49epYSfy4hfqQ2DW3zrU2cpgqAynlIFCUYp1tQLi1H80/CqUTCWurdwG17I4uZViQkckMJHqt9XwqdfskuOvjniOQWyjwsyr41qWOLsqEyJmKbxVSC3nSoaiCN7inZGctVVf3jz7ins8GcVh0l57oltuwQ5dKgUSUkpi6VExljq1keOccYZfeZ6DEOlpa0S5jXBJQY9xtCQJtoTrW99uXpGEXuh8pPc6hbZH+4elePe1bxXjHV7rUlZ71hCj8DVqKI8eo/vP3LrvtKiYRgsLqI6RLjp1IN3XVA6E6VEn2txAgtlpkakNMMo1BVqlE2teazxmO3L8En5rrqz6yPVKfgKdkQ5N8sPniePfEqexCk2CiXFBHVEmSSEjUSKlwxGEa6dZCsQsENpCgrIFaqKkkyTtBsntUkpoow6kttFbajJcUnsSopgmLoCgJBMSLixmh2OclYvygTOUATlkWsSfGaoQnnySVEydydydTWw/szf8AtHmt1t2vEKTJBna9YYK+tav+zZ5KcXmWoITlVKjoLUhM9Xa4likWCxaNTNdfxi3JUrW6TA7BQXGe1eCQJDhdi3UCIPjEUHxXtc7iPssJhj+aCsyecWHfNZyaNaWnqT/xWO/T34BXti+txxtxxQUooHhvFu+gKnbWAHaPhFEsVh1pAzuDpJUFNxJTH4the2/fVN5o+8UW5gx+hrz4xlTilUeTq1EqbqPY8fpwQdJpfapkukxBHn8qiJQQblP5kyPNP0pJYJuIV2pPyq3bqY2fQsNDMQmE3OqrDxmtCWWYH2YkJuRKdbGIgHf0rJrJQbz5fuatrfQgBTbpJvsUm/dI9aiUexcJJchNTbAMQ94KT80TSqo1xdUCVI/qEnxMXpVPmNd9MJkVxSqcaaqvTPPIXDVJ25Ig23Ohq+uoHE0mAGxWFFDwFIVmTY6dhHI8xWhebodiWKkZcwGOS6kpVy8UnTXcaX89jWbwxhObcfHSrISpBzpMFPz25HuO1dwbYWsAgdZ5sEbQokmAe6Ka5KcnaxO0l1sJLqFBp2yVEWnUQee8cqJF0qRc3Sf0Pyr0BngzGJwjjCbKcQlUric4nKq2hCiDF7BQ+8a8zwTpulViQQQdlCxHfao1EPLfsen8F1OyvsfEsfPp9B81e4O6pLqVJMESQfT51Nw7BsqaUpwqBnqqQAcsW64JFjrt3jQoYNbPXELR+NNwNPeGqfEd01xVaNTY2ux7sNdQnOVK+VdWfX8j0jgPtO9ZCgFfzCx+lHcR7SKAIy3IOvxmvMuFe0aEDrgpvObbz+WtEcZ7WsiyXW1E7gz6V4c9R8QU9qb9l+9jir6KhvuonoOEx6i2hSZugGOQ/ShnE8cX7ZLpkFU6+FAuH+2GEbZbQtL6lpTByNyPPwqFz24wdynCPL/OuP8AaTAr6qnPyJvmx89LT1HJqMXz2DLYKbkAD+ZUd8TFW8JCgEoCiSoSROUAGZzaHuFZL/r9Kf8ACwiEeR+EVF/13j3TDSUAckIKj61bnHuWtDXedr+eP3NB7esEMzplcYV5Oo+VeRe1gjEE/wAqfRKvpWvxX95Y8ltSXTHWyhAToREztMaVdf8AYh5RCnmm0kJAl59CdJ1yEk68qqLMZU3Hk8sUD8R6pFGMBgEtJ6Z4ToUNm+5UCoeIISbRdVoC90fZvDNXcxWEb1shtb3fBWQB5U3EcU4cg3fxb6uSOjbSf9KSr1q7GdzB4nB4nELzqbVciJmBuesbH829Nw3AXFHKCkq/Ck51X7Gwoite57SMA/YcObUfxPqW8e/rmB5VBjPa3HkZemSwj8LYS2B/pAVT2kuTKbHsM6AFOIUhP4nSlpP+4lUf01ZVg8KyjJ0iFnMnMGkqVp/3VwknsTlkxe1UsJw5/FHMhGIxJP3kJUR4uqgDxNEV8CdbQ6CrCMrbZW6WlOhx4hKSu6UykSkSJND29Rxck7ohbdb0awoWRPXfJXA/IISI5yahf4pEhzEkA/5bJt4pRCfM1mMTjVrspaiPw6AeAt6VC2CbJHkKV0uCm5PDYbe42BZtsAbFz5JFp86sYBRW3mWoqKiZ20gCBYR2UKZ4Yo3Vb1NEwiEgAC29ceqbnHamCXYsKSgQMsz2fT61G9gUm6bd37muBfZ+vfTEuXO/ftXmqnUWUxq64OFpaZykkR7qo9Kq5yPeZHh+zVtD8yKRgHTvg/uKtTlF2aNFUZUCmd1LHZFKrqWgfvKHcT8rUqvxody/FQXJimqNdJv9aauvUMCJVRxUjgPL1rkUrARLBqq6irik1EW6ABT7VR8Ms+yDp07R8M0fA0Rcboe9KCFfhUD5GaS5Gep4VsNulQBBPWB3CQcoH73rz/21wgZx+ISn3SvpEnmHAFn/AHFVepe07SOo6n7pUiRpDkOgHvzCPyGvMvbz/wDpbVzaT/5LrWWUTTk4yTXKBWGxakKCkKKT6HsNGsFxJBV738O7z/y1TzH3J7LHeayy1xY+FSoc2Nxy+h2qKbsrG+ptOfiLrn8n1NrhmWCvLiULZJiHGroMm5IHnon50df4fwtsSrGYhfY2n6ivPcBxBxAypOZP4Fbfl/Txq+1xVsjrIczcgRHrQ6UHmxUddqIrapuxqMN/dfXK04paZ6oJQDG+aCIvXFcb4Yj/AA+HFXa66fgCazBx3Jgx/Mo/IVXc4gRoGk+IPxV8qrwqfYh6zUPDm/dmwT7aNo/wuH4ZB2PRgnzNSOe3eOUITCB/KkD4CsIriSv/AFY/II+Q+NVHsUDqpavGPmapKK4RjKU5cs1uL9oMUtUuYggbyv5TQbFY1JJl5Su6f0FAy+NkjxUflFMGJVNiB3AfHWq3E7TS8H4YvElXQM58gBWpbiEJSDoVFR3ijbfswoNhxeKwqUlRQBhwvErKkgFSQhAuQCJjSRND/YtQ/h31LU2AcTggVOoLiRBfV1kAgrFtJr0TibTr38I8ynELShrEtqXh0jDErWWCCUvEFto5FGRJ6oid4c2PagIz7J4RLraHFYzEJUppK1goaQhTwCkJLdnSYUCQNAb0b9gcHgnSostYMDKFISnM483JIHSLckZtJA0Mi9D8U4wziEOYzG4VSkKYcK0qUt4KbaQ2tKW0CEBS0ElepSrLFhA7h/t/hcGMrAfxRQ2lptToQ0lCE6JGUZjsSpQkxU5Y7JBhKnscjDhWRz7VDLiHc3RKUxh3i8ooTAV9quIsCWxpFN9sGOjw2NSGS02jpShRCUoVmw6MMhDQmTYnQQAKyGL/ALRcYoFLAaw6SpSoaQJlZKlErVNySSSANaz2JW6+rM86paualFR8zpRYdwGGOdFeGYUpkxAI31NWmGEp0F+etT0nkQ2K7TqQRWbiFyM0xSam6OuZKnw0O5ARHdXMk7fsa+lTGuBA3NZukUskIdPM0q6UdtKsfAj2FtDGeu5qhFdmvQEdLhnemEx9aaVeNdJ8aAOTNMcEVzrToI7/ANK6QaQES6p4xuUkdlXVVGtNAG+4XjFv8MadauW0Ft9MjRvrIXfWAdRe/ZXn3tupYeQlz3kpTaQYBlQFuxVP4Hx97BLWG1AZhoRIIOxHnQHimMU67mUZUTJ2HgNgBbwq27om2bkeJ+vwqJjEhJyq0mx5fpXXDQ/Fqm1SWgy5YVE3jVi2dUcsxqvgMSCAhzTY/Kix4OkiUqt4U0JlA4jc+pppxNERw9san1FIpYTqUfGmK4LOIO011KVnRJogeIMJ0PkKhXxpGySe+kGSEYJw11XDT201zjSzokCqruMcVqo91K6HZmk9n+N4nAhwMP8ARBzLnskzlzRGYEp946VBxT2jceP22Idd7FLUoeRMDyrOZCdakS3S3DsWl478KY76SXFHU1EhFTtxS3MLFrDqIq827Q9Bqy09VAX0KqdJqk2uaspNIRaaSSQBcmw53pz7JQopUCFAwRuKrRT0Ki4oETrFNItt/wAU3pZvvXZvQMaoUxSKkJphosBx1lM9UkjmRHpJjzpVw0qQF7LypFNPz00j/irsAlI/fdXAmknXtrmSd459n1oAaqll2rmlNUPH9KQDVgVCRUylx+9ain98/wBKQFHGYXMI3GhoG7hloOk9taVV6ifQD20hmYXmPZUXQ0cdw1QKw9SMElqnBSojMY5TRAsRUfQ07gUVJO5pBurpbimdHQBV6OnhurGSnJRSEQdHTgipstdSigCMN04IqXLIroTQMjQmnpFPCacE0Ajg86lbFJKKmQJpgdbtVlC+dRIR+/3pUqU+NNMRYSakSrn361XBqUTJGo+FKTsOKuTtKbKFElWa2UAWOslRJkdkA0xBqJSADMd+hEUkOfv9ihSG4kyaUTqYpmf17fpXA6NN9p+XM09yFsY4gbgfvxpVwPjf0FKndBtZfIrsj9+dcpVRJ1VjUK3Ne6lSoAYpUX+Ncc+HPtrlKpYDQZ07aRnQUqVAEKyKasW05xXaVICBSahKKVKkxkbqJqIorlKkMa43TQ3SpUwOZaQTSpUANyUsvnSpUCHpbrhTypUqARLl5VKhGnPl+zSpUDHpbnapAmlSoETJmKeAKVKiIM6keN66kmu0qbEd1nxpyotrJ/fyrlKpZSZ2ByvXFNzJ2+H1rlKiSQ4t3GlvkTFdpUqyuzax/9k=`

Reply:
This is the only thing I didn't really fix.   I propose we kick it down the road till it's got a use